src/ol/geom/geometry.js

```diff
@@ -132,6 +132,7 @@ ol.geom.Geometry.prototype.getClosestPoint = function(point, opt_closestPoint) {
 /**
  * @param {ol.Coordinate} coordinate Coordinate.
  * @return {boolean} Contains coordinate.
+ * @api
```

Comment:
I'm in favor of exporting `containsCoordinate`.  But we need to have better docs when we export.  This needs at least a short, one sentence description.  With more detail if necessary.

It would be good to describe our interpretation of "contains" here.  A strict topological interpretation might say that ["polygons do not contain their boundary"](http://www.vividsolutions.com/jts/javadoc/com/vividsolutions/jts/geom/Geometry.html#contains(com.vividsolutions.jts.geom.Geometry)).  So a point that "touches" a polygon but is not in the interior would return false for `poly.containsCoordinate(point)`.